packages/components/time-select/src/time-select.ts

```diff
@@ -79,6 +79,13 @@ export const timeSelectProps = buildProps({
    * @description maximum time, any time after this time will be disabled
    */
   maxTime: String,
+  /**
+   * @description whether `end` is included in options
+   */
+  includeEndTime: {
+    type: Boolean,
+    default: true,
```

Comment:
Maybe we should set the default value to `false` and not change the original user's usage.

Reply:
Sure make sense